gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/baseline.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "org.hypertrace.gateway.service.v1.baseline";
+
+package org.hypertrace.gateway.service.v1.baseline;
+
+import "org/hypertrace/gateway/service/v1/gateway_query.proto";
+
+message BaselineEntitiesRequest {
+  string entity_type = 1;
+  sfixed64 start_time_millis = 2;
+  sfixed64 end_time_millis = 3;
+
+  repeated string entity_ids = 4;
+  repeated org.hypertrace.gateway.service.v1.common.FunctionExpression baseline_aggregate = 7;
```

Comment:
could we make the naming of these symmetric with the response - e.g. metric_request/metric, metric_series_request/metric_series or something similar (same for the types themselves)

Reply:
Done.